app/client/src/features/events/Questions/QuestionActions/QueueButton.tsx

```diff
@@ -29,8 +29,12 @@ export function QueueButton({ fragmentRef }: QueueButtonProps) {
 
     // TODO: add an animation for this using framer motion
     return isQueued ? (
-        <DequeueQuestionButton questionId={questionId} />
+        <div>
+            <DequeueQuestionButton questionId={questionId} />
+        </div>
```

Comment:
Is there a reason two divs are added here?

Reply:
without the divs they don't format correctly with the css I added to prevent them overlapping.

Reply:
Hmm, that sounds odd/doesn't sound right -- maybe it is, but I'll take a closer look and update this comment.

Reply:
Resolving since we established that the component needs a little bit of cleanup and is out of scope for this PR.